2.4/contrib/common.sh

```diff
@@ -98,7 +98,9 @@ function no_endpoints() {
 function build_mongo_config() {
   local members="{ _id: 0, host: \"$(mongo_addr)\"},"
   local member_id=1
+  local container_addr="$(container_addr)"
   for node in $(endpoints); do
+    [ "$node" == container_addr ] && continue
```

Comment:
Since we run this from a post deploy hook pod, we don't want to include it's IP in the replica set config.